providers/class.two-factor-fido-u2f.php

```diff
@@ -23,12 +41,38 @@ static function get_instance() {
 	}
 
 	/**
+	 * Class constructor.
+	 *
+	 * @since 0.1-dev
+	 */
+	protected function __construct() {
```

Comment:
It may be worth adding in an extra `if ( version_compare( PHP_VERSION, '5.3.0', '<' ) ) return;` or something in here just as a backup to avoid fatal errors.